src/components/WebexDataProvider/README.md

```diff
@@ -1,3 +1,29 @@
-# Webex Activity Data Provider Component
+# Webex Data Provider
 
-LALLI HELP ME HERE PLEASE. MY BRAIN IS FRIED RIGHT NOW!
+Webex Data Provider creates a context that providers data adapters to its child components.
+
+## Usage
+
+1. Create an adapter for a data source.
+Webex Componets come with a JSON data adapter used to test components with static data.
+
+    ```js
+    import {WebexJSONAdapter} from '@webex/components';
+    import jsonData from './my-data';
+
+    const jsonAdapter = new WebexJSONAdapter(jsonData);
+    ```
+
+2. Create the data provider and give it the previously created [data adapter](../../adapters/WebexJSONAdapter.js).
+Code should look something like this:
```

Comment:
We could remove this line since the example in step 1 just goes directly into the sample code for consistency.

Reply:
Not sure I understand which line you're referring to?

Reply:
```suggestion
Code should look something like this:
```

Reply:
This statement: `Code should look something like this:`. The sample code is implied on the step above